src/lwc/gsResources/gsResources.js

```diff
@@ -0,0 +1,70 @@
+import { LightningElement } from 'lwc';
+import gsResourcesTitle from '@salesforce/label/c.gsResourcesTitle';
+import gsResourcesItemNonprofitCloudLabel from '@salesforce/label/c.gsResourcesItemNonprofitCloudLabel';
+import gsResourcesItemNonprofitCloudLink from '@salesforce/label/c.gsResourcesItemNonprofitCloudLink';
+import gsResourcesItemFundraisingLabel from '@salesforce/label/c.gsResourcesItemFundraisingLabel';
+import gsResourcesItemFundraisingLink from '@salesforce/label/c.gsResourcesItemFundraisingLink';
+import gsResourcesProgramManagementLabel from '@salesforce/label/c.gsResourcesItemProgramManagementLabel';
+import gsResourcesProgramManagementLink from '@salesforce/label/c.gsResourcesItemProgramManagementLink';
+import gsResourcesMarketingEngagementLabel from '@salesforce/label/c.gsResourcesItemMarketingEngagementLabel';
+import gsResourcesMarketingEngagementLink from '@salesforce/label/c.gsResourcesItemMarketingEngagementLink';
+import gsResourcesWebinarsLabel from '@salesforce/label/c.gsResourcesItemWebinarsLabel';
+import gsResourcesWebinarsLink from '@salesforce/label/c.gsResourcesItemWebinarsLink';
+import gsResourcesHubLabel from '@salesforce/label/c.gsResourcesItemHubLabel';
+import gsResourcesHubLink from '@salesforce/label/c.gsResourcesItemHubLink';
+import gsResourcesPartnersLabel from '@salesforce/label/c.gsResourcesItemPartnersLabel';
+import gsResourcesPartnersLink from '@salesforce/label/c.gsResourcesItemPartnersLink';
+
+export default class gsResources extends LightningElement {
+
+    get title() {
+        return gsResourcesTitle;
+    }
+
+    get resources() {
+        return [
+            {
+                id: 1, 
+                label: gsResourcesItemNonprofitCloudLabel, 
+                linkLabel: gsResourcesItemNonprofitCloudLink,
+                href: 'https://trailhead.salesforce.com/en/content/learn/modules/nonprofit-cloud-basics'
```

Comment:
Can't we leave the links in CustomLabels so it is easier to update in the future?